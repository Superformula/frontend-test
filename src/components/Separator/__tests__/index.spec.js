import React from 'react';
import { mount } from 'enzyme';
import { Separator } from '../';

describe('Separator', () => {
  it('should render without failing', () => {
    const wrapper = mount(<Separator/>);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render a spaced separator', () => {
    const wrapper = mount(<Separator spaced/>);
    expect(wrapper).toMatchSnapshot();
  });
});
