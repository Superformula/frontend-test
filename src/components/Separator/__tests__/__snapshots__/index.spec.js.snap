// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator should render a spaced separator 1`] = `
.c0 {
  border: 0;
  border-top: 1px solid #E6E6E6;
  margin-top: 24px;
  margin-bottom: 24px;
}

<Separator
  spaced={true}
>
  <hr
    className="c0"
  />
</Separator>
`;

exports[`Separator should render without failing 1`] = `
.c0 {
  border: 0;
  border-top: 1px solid #E6E6E6;
}

<Separator>
  <hr
    className="c0"
  />
</Separator>
`;
